Automattic/wp-calypso
# Add VideoPress to Complete features list

### Changes proposed in this Pull Request

This PR adds _VideoPress_ to the list of features included in the _Complete_ plan. Consequently, the _VideoPress_ product card is marked as included when visiting the pricing page while the selected site has a _Complete_ subscription.

Fixes 1164141197617539-as-1201541030589496

### Testing instructions

- Download the PR and run Calypso or cloud
- Select a self-hosted site with a _Complete_ subscription
- Visit the plans/pricing page with this site selected (in cloud, `https://cloud.jetpack.com/pricing/:site`)
- Check that the _VideoPress_ card is marked as included

### Screenshots

| Before | After |
|--------|------|
|<img width="352" alt="Screen Shot 2021-12-20 at 3 15 02 PM" src="https://user-images.githubusercontent.com/1620183/146827624-e5e3cc41-332a-45a5-8a0b-bc07b9179f69.png">|<img width="352" alt="Screen Shot 2021-12-20 at 3 14 39 PM" src="https://user-images.githubusercontent.com/1620183/146827645-84a3908f-423a-4f16-9635-3c56bd7c9b84.png">|
